docs/app/components/copy-button.css

```diff
@@ -0,0 +1,11 @@
+.copy-button{
+  background-color:transparent
+}
+
+button{
```

Comment:
let's use class name instead of html tag selector, also add an empty line at the end